run/multibit2john.py

```diff
@@ -1344,22 +1344,18 @@ def process_file(filename):
                         return
             return
 
-    version = 1  # MultiBit Classic
     pdata = b"".join(data.split())
     if len(pdata) < 64:
         sys.stderr.write("%s: Short length for a MultiBit wallet file!\n" % bname)
         return
+    pdata = base64.b64decode(pdata[:64])
```

Comment:
IIRC this can throw an exception. The `try` block is to catch exceptions from this line.

I vaguely remember that I couldn't find a way to distinguish between MultiBit wallet versions 100%. Without finding such `distinguishers`, things become tricky - hence the warning `Make sure that this is a MultiBit HD wallet!`.

I might be wrong about all this as it has been forever since I wrote this ;(

Reply:
Hi Dhiru! Oh, right, thanks. I just considered the already-tested length - but maybe the actual data isn't always Base-64... I will amend this PR.